frontend/packages/process-editor/src/components/Canvas/Viewer/Viewer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode } from 'react';
+import classes from './Viewer.module.css';
+import { useBpmnViewer } from '../../../hooks/useBpmnViewer';
+import { VersionAlert } from './VersionAlert';
+import { ViewerErrorAlert } from './ViewerErrorAlert';
+import { getIfVersionIs8OrNewer } from '../../../utils/processEditorUtils';
+
+export type ViewerProps = {
+  appLibVersion: string;
+};
+
+/**
+ * @component
+ *  Displays the canvas area of the ProcessEditor
+ *
+ * @property {string}[appLibVersion] - The app-lib version the user has
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const Viewer = ({ appLibVersion }: ViewerProps): ReactNode => {
```

Comment:
I think we should rename this to BPMNViewer since it is a separated component now. 😊